frontend/src/pages/BlogSingle.jsx

```
@@ -131,7 +132,7 @@ const BlogSingle = () => {
                   lineHeight: '1.8',
                   fontSize: '1.1rem'
                 }}
-                dangerouslySetInnerHTML={{ __html: blog.content }}
+                dangerouslySetInnerHTML={{ __html: blog.content.replace(/\n/g, "<br/>") }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)